### ISSUE TITLE
An easy way to use hatching in polygons

### ISSUE THREAD

**Issue by indapublic** [mmomtchev/rlayers#129]
First I want to thank you for the library. Simply incredible.

I have a question: it is necessary to hatch polygons according to a given pattern ([example](http://viglino.github.io/ol-ext/examples/style/map.style.pattern.html))

Is there any easy way to use `ol-ext` with your library? 
Or is this functionality already available (but didn't find it)?

Thank you

**Comment by indapublic** [comment]
[Did](https://github.com/indapublic/rlayers/commit/17610d2ebd32e85c472b7035c74912b23b6249f4) it, but I really hope there is a better way to do this.

**Comment by mmomtchev** [comment]
No, I don't see how you could reduce the boilerplate code in this case? You have to have a separate component type that wraps this feature. I have tried to extract all the common logic in the base class, but you still have to enumerate the properties.

Otherwise, there is no support for any `ol-ext` features - maybe having an additional package should be the way to go for those.
